### ISSUE TITLE
LICENSE.txt missing in tarball on pypi.org

### ISSUE THREAD

**Issue by dvzrv** [hsluv/hsluv-python#28]
Hi! I'm packaging hsluv for Arch Linux.
To be able to distribute the package, I require the license file, that is contained in this repository.
Unfortunately it is not contained in the source tarball on pypi.org

**Comment by hartwork** [comment]
The text is standard MIT license text. I'm packaging downstream myself, for Gentoo in my case. Why do you need the file in packaging?

On the technical part, it seems that [file `MANIFEST.in`](https://github.com/hsluv/hsluv-python/blob/master/MANIFEST.in) is out of sync, and maybe should be deleted altogether.

**Comment by dvzrv** [comment]
The MIT license is specific to each project, as [it states the person/entity holding the copyright](https://github.com/hsluv/hsluv-python/blob/master/LICENSE.txt#L1).

These license files need to be present when distributing software in source or binary form. For other licenses (e.g. GPL) it is sufficient to have one copy of the license file on the system, as the license file is not unique per project (does not state the entity holding the copyright). This is at least according to our current packaging guidelines on Arch Linux.

**Comment by dvzrv** [comment]
> On the technical part, it seems that [file `MANIFEST.in`](https://github.com/hsluv/hsluv-python/blob/master/MANIFEST.in) is out of sync, and maybe should be deleted altogether.

Yes, amending that file would fix this and #29.

**Comment by hartwork** [comment]
> The MIT license is specific to each project, as [it states the person/entity holding the copyright](https://github.com/hsluv/hsluv-python/blob/master/LICENSE.txt#L1).
> 
> These license files need to be present when distributing software in source or binary form. For other licenses (e.g. GPL) it is sufficient to have one copy of the license file on the system, as the license file is not unique per project (does not state the entity holding the copyright). This is at least according to our current packaging guidelines on Arch Linux.

I follow your explanation but that seems way out of proportion, even with me caring about licensing in general. For comparison, there is a single MIT license in Gentoo :

```console
# head -n1 /usr/portage/licenses/MIT
Copyright (c) <year> <copyright holders>
```

I would normally ask for a reference/source on that statement with regard to Arch but since we have a bug in `MANIFEST.in` here anyway, I'll take that as-is and open a pull request. Give me a few minutes.
